sonaranalyzer-dotnet/its/sources/Net5/NativeInts.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Net5
+{
+    public class NativeInts
+    {
+        public void Method()
+        {
+            nint i = -1;
+            nuint i2 = 2;
+            Console.WriteLine($"Type of {i} is {typeof(nint)}."); // System.IntPtr.
+            Console.WriteLine($"Type of {i2} is {typeof(nuint)}."); // System.UIntPtr
+        }
```

Comment:
According the the proposal

```
    enum E : nint
    {
    }
```

this should be permitted but currently it is not. Not sure if we should add a test case (it will be commented but we will not forget about this)